python/strings_udf/strings_udf/tests/test_string_udfs.py

```diff
@@ -68,7 +69,8 @@ def run_udf_test(data, func, dtype):
     str_views = column_to_string_view_array(cudf_column)
 
     kernel = get_kernel(func, dtype, len(data))
-    kernel.forall(len(data))(str_views, output)
+    with acquire_spill_lock():
+        kernel.forall(len(data))(str_views, output)
```

Comment:
```suggestion
    kernel.forall(len(data))(str_views, output)
```
Dropping adding of spill lock here for what @shwina brought up and consistency with cudf pytests too.